staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -48,6 +48,14 @@ const (
 	DiskErrorPolicyEnospace DiskErrorPolicy = "enospace"
 )
 
+type PanicDeviceModel string
+
+const (
+	PanicDeviceModelHyperv  PanicDeviceModel = "hyperv"
+	PanicDeviceModelIsa     PanicDeviceModel = "isa"
+	PanicDeviceModelPvpanic PanicDeviceModel = "pvpanic"
```

Comment:
Would you mind dropping the repeated type of `PanicDeviceModel` from each of these constants assuming the resulting names don't conflict with anything?

https://libvirt.org/formatdomain.html#panic-device also suggests `s390` as a valid model, we do support `s390x` so I think we should add that here.

Reply:
```
This feature is always enabled for:
* pSeries guests, since it's implemented by the guest firmware
* S390 guests, since it's an integral part of the S390 architecture
For the guest types listed above, libvirt automatically adds a panic element to the domain XML.
```
Both pseries and s390 have this feature always enabled by libvirt. We wouldnt need to explicitly add them. Additionally, libvirt doesn't allow you to specify even an address for them:
```
Most users don't need to specify an address, and doing so is forbidden altogether for s390, pseries and hyperv models.
```

Reply:
> Would you mind dropping the repeated type of PanicDeviceModel from each of these constants assuming the resulting names don't conflict with anything?
I can't drop the type here. go is inferring the other consts as strings, if i do so.

Reply:
ohh got it! I misunderstood the ask earlier.
 I've removed the type prefix from the constants now.